chemprop/args.py

```diff
@@ -412,12 +478,16 @@ class TrainArgs(CommonArgs):
     """Maximum magnitude of gradient during training."""
     class_balance: bool = False
     """Trains with an equal number of positives and negatives in each batch."""
-    spectra_activation: Literal['exp', 'softplus'] = 'exp'
+    spectra_activation: Literal["exp", "softplus"] = "exp"
     """Indicates which function to use in dataset_type spectra training to constrain outputs to be positive."""
     spectra_target_floor: float = 1e-8
     """Values in targets for dataset type spectra are replaced with this value, intended to be a small positive number used to enforce positive values."""
     evidential_regularization: float = 0
     """Value used in regularization for evidential loss function. Value used in literature was 1."""
+    quantile: float = 0.5
+    """Quantile for quantile loss"""
```

Comment:
this quantile arg doesn't seem to actually be used? It looks like it's overwritten at the beginning of crossvalidation.py.

Reply:
Hey Charles, your right, I think this is legacy from when I had originally implemented pinball loss.